gflib/malloc.c

```diff
@@ -1,8 +1,11 @@
 #include "global.h"
+#include "malloc.h"
 
 static void *sHeapStart;
 static u32 sHeapSize;
 
+__attribute__((section("__EWRAM_HEAP"))) u8 gHeap[HEAP_SIZE] = {0};
```

Comment:
As an aside, I don't think there's a technical reason to have the heap at the start of EWRAM. For matching purposes you can link `malloc` at the start of `sym_ewram` rather than giving `gHeap` its own section, and let modern throw the heap wherever.